test/freenet/client/filter/CSSParserTest.java

```diff
@@ -9,13 +11,10 @@
 import java.io.StringWriter;
 import java.net.URI;
 import java.net.URISyntaxException;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Iterator;
-import java.util.LinkedHashMap;
+import java.util.*;
```

Comment:
I am confused. In another test a star import was expanded, and here you’re collapsing it again?

Reply:
I think it was some automatic effort of IDE.
Expanded it back.